src/renderers/shared/fiber/ReactFiberCommitWork.js

```diff
@@ -221,13 +222,27 @@ module.exports = function<T, P, I, TI, C>(config : HostConfig<T, P, I, TI, C>) {
     }
   }
 
-  function commitUnmount(current : Fiber) : void {
+  function commitUnmount(current : Fiber, safely : boolean) : void {
+    // Make sure we mark this deletion as completed so that we don't
+    // attempt to perform it again if one of the deletions fails, and
+    // error boundary tries to unmount this subtree again.
```

Comment:
Can you explain how this happens? When we commit, we've already swapped the tree so as far as Fiber is concerned, it is deleted in the "current" state. So any rerender should presumably not try to unmount again.